pkg/generated/lister/core/v1alpha1/zfssnapshot.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright 2019 The OpenEBS Authors
```

Comment:
2020 :-)

Reply:
autogen code, can not change the boiler-code.txt to 2020 as it will regenrate the whole lister/informer to 2020 and we have changed snapshot code only? Should I regenerate it with 2019-2020 Copyright?